packages/io/stl-deserializer/test/test.js

```diff
@@ -1,20 +1,22 @@
 const fs = require('fs')
 const path = require('path')
 const test = require('ava')
+
+const { geometry } = require('@jscad/csg')
+
 const deserializer = require('../index.js')
 
 const samplesPath = path.dirname(require.resolve('@jscad/sample-files/package.json'))
 
-const polygonsFromCsg = csg => csg.polygons.map(x => x.vertices.map(vert => ([vert.pos.x, vert.pos.y, vert.pos.z])))
-// x=>console.log(
+const toArray = (polygons) => polygons.map((p) => p.vertices.map((v) => ([v[0], v[1], v[2]])))
+
 test('translate simple ascii stl to jscad code', function (t) {
   const inputPath = path.resolve(samplesPath, 'stl/testcube_ascii.stl')
   const inputFile = fs.readFileSync(inputPath, 'utf8')
 
-  const expected = `function main() { return union(
-// objects: 1
+  const expected = `function main() { return [
```

Comment:
This also needs the require() calls and the module.export = {main} call at the end :)

Reply:
@z3dev This is one is very important, the implicit imports are gone from V2, so any piece of code using the api needs to require() what it needs explicitely

Reply:
Done